test/evergreen.yml

```diff
@@ -349,12 +349,13 @@ functions:
   "make check directory":
     command: shell.exec
     params:
-      working_dir: "wiredtiger/build_posix"
+      working_dir: "wiredtiger"
       script: |
         set -o errexit
         set -o verbose
-
-        ${test_env_vars|} ${make_command|make} VERBOSE=1 check -C ${directory}  ${smp_command|} 2>&1
+        . test/evergreen/find_cmake.sh
+        cd cmake_build
+        ${test_env_vars|} cd ${directory};$CTEST ${smp_command|} --output-on-failure 2>&1
```

Comment:
Can `cd ${directory}` be moved up (maybe combine with the previous cd command, so that `${test_env_vars|}` is guaranteed to apply to the $CTEST command?

Reply:
Btw, will this change lose the explicit setting of `VERBOSE=1`?

Reply:
Good point, I've reworked the ordering of cd commands here: 175f33f5831dac971fd223b58e7d4d57eb0b48e8

---

Given CTest is already quite verbose, sometime ago we removed uses of the verbose option `-V` (equivalent to `VERBOSE=1`), in favor of `--output-on-failure`. This option will dump the entire log of a test if it fails, otherwise it masks the output to just summary information. My intention was to maintain that setting here.